for_anas.py

```
@@ -0,0 +1,181 @@
+
+import json
+import requests
+
+# ========== Stocks from Analytic (with paginations)========== 
+def stock_from_analytic(header, offset=0, limit=1000, attempt=1, data_list=None):
+    """
+    Остатки товаро из аналитики
+    """
+    if not data_list:
+        data_list = []
+    url = f'https://api-seller.ozon.ru/v2/analytics/stock_on_warehouses'
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)